common/autoware_auto_perception_rviz_plugin/include/object_detection/object_polygon_detail.hpp

```diff
@@ -80,7 +80,7 @@ AUTOWARE_AUTO_PERCEPTION_RVIZ_PLUGIN_PUBLIC visualization_msgs::msg::Marker::Sha
 get_shape_marker_ptr(
   const autoware_auto_perception_msgs::msg::Shape & shape_msg,
   const geometry_msgs::msg::Point & centroid, const geometry_msgs::msg::Quaternion & orientation,
-  const std_msgs::msg::ColorRGBA & color_rgba);
+  const std_msgs::msg::ColorRGBA & color_rgba, const double line_width = 0.03);
```

Comment:
I am confusing why we have to specify default value here. (But actually rviz crush without default value.)

Reply:
We didn't need to specify a value, but 0.03 was originally used as line_width, so I set it as default value.
![image](https://user-images.githubusercontent.com/59680180/177305754-d6f5e5d0-c346-4f32-bd68-689868c90382.png)

I removed default value at [652a33a](https://github.com/autowarefoundation/autoware.universe/pull/1206/commits/652a33a8bf3040a7963f820fa8fc5868e54504d6) and confirmed that it works well without crushing

Reply:
I tried again and rviz does not crush.
So [652a33a](https://github.com/autowarefoundation/autoware.universe/pull/1206/commits/652a33a8bf3040a7963f820fa8fc5868e54504d6) looks good for me!